rpi3.mk

```diff
@@ -184,15 +185,41 @@ rpi3-firmware:
 ifeq ("$(wildcard $(ROOT)/out/$(RPI3_FIRMWARE_FILE).$(RPI3_FIRMWARE_FILE_EXT))","")
 	echo "Downloading Raspberry Pi 3 firmware ..."
 	mkdir -p $(ROOT)/out
+	mkdir -p $(RPI3_STOCK_FW_PATH)/boot
 	wget $(RPI3_FIRMWARE_URL)/$(RPI3_FIRMWARE_FILE).$(RPI3_FIRMWARE_FILE_EXT) -O $(ROOT)/out/$(RPI3_FIRMWARE_FILE).$(RPI3_FIRMWARE_FILE_EXT)
-	unzip -a $(ROOT)/out/$(RPI3_FIRMWARE_FILE).$(RPI3_FIRMWARE_FILE_EXT) -d $(ROOT)
-	mv $(ROOT)/firmware-$(RPI3_FIRMWARE_FILE) $(RPI3_STOCK_FW_PATH)
+	unzip -aj $(ROOT)/out/$(RPI3_FIRMWARE_FILE).$(RPI3_FIRMWARE_FILE_EXT) \
+	'$(RPI3_STOCK_ZIP_BOOT)/bootcode.bin' \
```

Comment:
@jenswi-linaro 
> This list occurs when removing also, perhaps it should go into a variable to save the repetition.

The main issue that $(RPI3_STOCK_ZIP_BOOT) != $(RPI3_STOCK_FW_PATH_BOOT), so if we are going to use a variable with list of files we have to append proper path in some foreach loop and call unzip for each file; or save "unzip" commandline to another variable and costruct it by appending each file, and then invoke it. Anyway, it will just add additional complexity.